ansible/inventory.yml

```diff
@@ -74,7 +74,7 @@ hosts:
   - test:
 
     - azure:
-        msft-ubuntu1404-x64-1: {ip: node-msft-cross-compiler-1.cloudapp.net}
+        msft-ubuntu1604_arm_cross-x64-1: {ip: nodejs.eastus2.cloudapp.azure.com, user: ubuntu}
```

Comment:
the naming is non-standard btw, should remove "msft" from here since "azure" goes into the name already and the pattern is supposed to be: "type-os/dist+version[_extrainfo]-arch-X"

Reply:
I would like the name to remain `azure_msft`. Azure is where the machines are hosted, however it is not Azure that is sponsoring us but Microsoft directly.

There are 2 machines that have only `msft`, because they are physically located in my office. They're meant to be only temporary until we can get ARM64 Azure VMs and I can't guarantee they'll remain here, so just `msft` is fine for those 2.

This is covered in our current documentation: https://github.com/nodejs/build/tree/master/ansible#naming , I took that to mean `provider_sponsor` when the two are different, as with the Raspberry Pi's.

Reply:
oh yeah, you're right, all good then